src/core/tests/util/graph_comparator.hpp

```diff
@@ -800,6 +837,36 @@ struct Get<ov::op::util::FrameworkNodeAttrs, void> {
     }
 };
 
+template <>
+struct Get<ov::Dimension, void> {
+    static std::string value(const ov::Dimension& attrs) {
+        std::stringstream oss;
+
+        if (attrs.is_dynamic() && !attrs.get_interval().has_upper_bound() && (attrs.get_min_length() == 0))
+            return "[dynamic_dim]";
```

Comment:
Why we don't reuse this logic from Serialization? https://github.com/openvinotoolkit/openvino/pull/8859/files#diff-6a91cb0e5781b477826c26a472e31e77a27e2dc090110161c1d97c2b72da9b03R49 looks like a duplicate.

Reply:
Used operator<< in this method.